### DISCUSSION TITLE
Проблема с отображением параметра выносливости у артефактов

### DISCUSSION THREAD

**Discussion by mole51venomous** [OGSR/OGSR-Engine#381] [Q&A]
Доброго времени суток. Впервые публикую сообщение на данном ресурсе и могу по незнанию оставить его в "неположенном месте". Прошу это принять во внимание.
После обновления своей сборки до версии движка движка OGSR 2.1.320 случайно обнаружил следующую проблему.
Артефакты, оказывающее воздействие на восстановление выносливости ГГ (power_restore_speed) имеют некорректное
отображение этого параметра в игре. Т.е. данный показатель продолжает работать как и раньше, но отображение с прямо противоположным знаком, при тех же цифрах. Отпугивающие красные цифры у арта "Лунный свет", замеченные мной, работают как и раньше, как зелёные. На предшествующей ревизии движка (2.1.293) этого явления не замечал.
В звязи с отсутствием опыта правок движка сам "внутрь" лезть нее стал.

**Comment by I-AmDead** [comment]
![ss_user_12-05-21_15-25-30_(l08_yantar)](https://user-images.githubusercontent.com/47627453/144739369-2263e8a7-34f6-4001-8e6f-59cb767f723f.jpg)

Готов это опровергнуть...

**Comment by mole51venomous** [comment]
Выходит, проблема лишь у меня, хоть движ я и не ковырял...
https://disk.yandex.ru/i/c_MO_tPvyJpP-A
На всякий случай, видео с тестом
https://disk.yandex.ru/i/PsZNMIyUVrVAdw

**Reply by I-AmDead** [reply]
> Выходит, проблема лишь у меня, хоть движ я и не ковырял... https://disk.yandex.ru/i/c_MO_tPvyJpP-A На всякий случай, видео с тестом https://disk.yandex.ru/i/PsZNMIyUVrVAdw

Выходит что так, у меня ни на одной версии движка такого не было

**Reply by mole51venomous** [reply]
> Перепроверил. Проблема идёт с версии 2.1.181, т.е. с первой работоспособной сейчас. С самой первой, использовавшейся мной 2.1.63 нынче не запускается. На сборке с движком 1.0006 + шейдеры OGSE всё красиво. Какая то случайная залётная правка, ну да бог с ней.
